docs/content/latest/integrations/prisma.md

```diff
@@ -5,15 +5,17 @@ description: Prisma
 menu:
   latest:
     identifier: prisma
-    parent: graphql
-    weight: 583
+    parent: integrations
+    weight: 579
 isTocNested: true
 showAsideToc: true
 ---
 
-Explore how you can use Prisma, and its GraphQL support, to interact with YugabyteDB. You will quickly build a GraphQL server and then use the Prisma client to write data to and run queries on a YugabyteDB database. Also, you'll get a taste of Prisma's ORM functionality.
+Explore how you can use Prisma and its GraphQL support to interact with YugabyteDB. You can build a GraphQL server and then use the Prisma client to write data to and run queries on a YugabyteDB database. Also, you can explore the Prisma's ORM functionality.
```

Comment:
Taking out the "quickly" changes the flavour of this one a bit more than we might want. I think we probably want to convey that it's quick and easy to get this set up; "you can build a GraphQL server" does not make me think it's going to be easy, from what I've experienced of GraphQL. 😆

Reply:
quickly, simply, easily, etc is marketing. When I see these things, I usually delete them from tech docs.